src/Jenssegers/Mongodb/Eloquent/Model.php

```diff
@@ -233,11 +234,11 @@ public function getAttribute($key)
 
                 // This attribute matches an embedsOne or embedsMany relation so we need
                 // to return the relation results instead of the interal attributes.
-                if ($relations instanceof EmbedsOneOrMany) {
+                if ($relations instanceof EmbedsOneOrMany or $relations instanceof MorphTo) {
```

Comment:
This seems a bit odd, this is very specific to embedded relations, why was `MorpTo` added here?